### PR TITLE
4040 Organization request from get to post, prevent possible Null Reference Exception

### PR DESCRIPTION
**Pull request by Vectorfield4**

### CODE REVIEW

**Comment by lnetrebskii** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-2185 has been created

File: VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs [hunk 1]
```diff
@@ -351,14 +351,14 @@ public IHttpActionResult GetOrganizationById(string id, [FromUri]string response
...
 
         /// <summary>
-        /// Get plenty organizations 
+        /// Get plenty organizations
         /// </summary>
         /// <param name="ids">Organization ids</param>
         /// <param name="responseGroup">Response group flags controls fullness of resulting object graph</param>
-        [HttpGet]
-        [Route("organizations")]
+        [HttpPost]
```
Comment (yecli) [inline VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs:358]: Agreed on the previous comment, we could not remove the existing method from API to save backward compatibility.
You can add POST, not to replace the existing one.

File: VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs [hunk 2]
```diff
@@ -351,14 +351,14 @@ public IHttpActionResult GetOrganizationById(string id, [FromUri]string response
...
+        [Route("get-organizations")]
```
Comment (yecli) [inline VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs:359]: Route("organizations/plenty") is more restful and aligned with the existing routing principles, e.g. [products/plenty](https://github.com/VirtoCommerce/vc-module-catalog/blob/dev/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleProductsController.cs#L101)

**Comment by yecli** [comment]
Released: https://github.com/VirtoCommerce/vc-module-customer/releases/tag/v2.14.27

**Pull request merged**